src/overlays/actors/ovl_En_Fg/z_en_fg.h

```diff
@@ -2,6 +2,7 @@
 #define Z_EN_FG_H
 
 #include "global.h"
+#include "objects/object_fr/object_fr.h"
```

Comment:
With `#include "objects/object_fr/object_fr.h"` in the header files now, could you remove this in both `fg.c` and `minifrog.c`?